include/gba/io_reg.h

```diff
@@ -599,6 +599,7 @@
 #define BLDCNT_EFFECT_BLEND     (1 << 6)   // 1st+2nd targets mixed (controlled by BLDALPHA)
 #define BLDCNT_EFFECT_LIGHTEN   (2 << 6)   // 1st target becomes whiter (controlled by BLDY)
 #define BLDCNT_EFFECT_DARKEN    (3 << 6)   // 1st target becomes blacker (controlled by BLDY)
+#define BLDCNT_EFFECT_EFF_MASK  (3 << 6)    // mask to check effect
```

Comment:
```suggestion
#define BLDCNT_EFFECT_EFF_MASK  (3 << 6)   // mask to check effect
```
Shouldn't this be `(4 << 6)` anyway?

Reply:
I don't know enough about these things to know if it should be `(4 << 6)` or not.
It compiles and the overworld is tinted, but I don't know.

Reply:
Me neither, but the context makes it seem like it shouldn't be the same as the darken effect. Maybe @aarant can check and confirm?

Reply:
Looking at it, it is correct.
It's named "MASK", and it behaves like a mask in code, it's used to mask off the first 2 bits for checking values.
Which makes sense.